Automattic/jetpack
# Update woo commerce recommendation illustration

This PR adds a new illustration for the woocommerce recommendation. Previously, it was using the default illustration, which is the same as the site-type recommendation question. This was the only recommendation that had this issue

#### Changes proposed in this Pull Request:

Add a new woo commerce recommendation illustration and use it

#### Other information:


#### Jetpack product discussion

Asana: 1198218726984184-as-1203032341480532/f

#### Does this pull request change what data or activity we track or use?

No it does not

#### Testing instructions:

1. Checkout this branch via `git switch update/woo-commerce-recommendation-illustration`
2. Go to `/wp-admin/admin.php?page=jetpack#/recommendations/woocommerce`
3. Confirm the new illustration is present
![image](https://user-images.githubusercontent.com/65001528/192634258-7a10c3ee-1d77-43b3-a9c4-a7ef61b0f742.png)
4. Make sure it looks alright on mobile and desktop
